release-notes/opensearch-security-dashboards-plugin.release-notes-1.3.2.0.md

```diff
@@ -0,0 +1,8 @@
+## Version 1.3.2.0
+
+### Bug Fixes
+* Fix 'openserach' typo in constants.tsx (#953) ([#959](https://github.com/opensearch-project/security-dashboards-plugin/pull/959))
+
+### Maintenance
+* Incremented version to 1.3.2. ([#978](https://github.com/opensearch-project/security-dashboards-plugin/pull/978))
+* Remove redundant DCO check in favor of the GitHub app ([#972](https://github.com/opensearch-project/security-dashboards-plugin/pull/972))
```

Comment:
We should add this PR here

Reply:
Nope, it isn't useful for readers of the release notes